### PR TITLE
Feature/eco bridge fixes

### PR DESCRIPTION
**Pull request by mix1o**
Post review amendments

### CODE REVIEW

File: src/services/EcoBridge/EcoBridge.types.ts [hunk 1]
```diff
@@ -55,7 +55,13 @@ export interface TokenMap {
...
   }>
 }
-export type AsyncState = 'idle' | 'loading' | 'ready' | 'failed'
+
+export enum AsyncState {
+  IDLE = 'idle',
+  LOADING = 'loading',
+  READY = 'ready',
+  FAILED = 'failed'
+}
```
Comment (adamazad) [inline src/services/EcoBridge/EcoBridge.types.ts:64]: SyncState maybe?

File: src/services/EcoBridge/EcoBridgeProvider.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+import React, { useContext, useEffect, useState, FC } from 'react'
 import { EcoBridge } from './EcoBridge'
 import store from '../../state'
 import { useActiveWeb3React } from '../../hooks'
 import { ecoBridgeConfig } from './EcoBridge.config'
 
 export const EcoBridgeContext = React.createContext<EcoBridge | null>(null)
 
-export const EcoBridgeProvider = ({ children }: { children?: React.ReactNode }) => {
+export const EcoBridgeProvider: FC = ({ children }: { children?: React.ReactNode }) => {
```
Comment (adamazad) [inline src/services/EcoBridge/EcoBridgeProvider.tsx:9]: I think `FC` add `children` property

File: src/services/EcoBridge/Socket/SocketBridge.ts [hunk 3]
```diff
@@ -27,9 +27,10 @@ import {
   DAI_ETHEREUM_ADDRESS,
   SOCKET_NATIVE_TOKEN_ADDRESS,
   WETH_GNOSIS_ADDRESS
-} from './Socket.types'
+} from '../../../constants'
 import { getBestRoute, getDAIAddress, getStatusOfResponse } from './Socket.utils'
 import { SOCKET_TOKENS } from './Socket.lists'
+import { BigNumber } from 'ethers/lib/ethers'
```
Comment (adamazad) [inline src/services/EcoBridge/Socket/SocketBridge.ts:33]: This import should be either:

```typescript
import { BigNumber } from 'ethers'
```

or 

```typescript
import { BigNumber } from '@ethersproject/bignumber'
```

Pick whichever you like.

File: src/services/EcoBridge/Arbitrum/ArbitrumBridge.selectors.ts [hunk 4]
```diff
@@ -119,7 +119,7 @@ const createSelectBridgeTxsSummary = (
       }
       if (!tx.partnerTxHash || !l2Txs[tx.partnerTxHash]) {
         if (tx.type === 'deposit-l1' && tx.receipt?.status !== 0) {
-          summary.status = 'pending' // deposits on l1 should never show confirmed on UI
+          summary.status = 'pending' // deposits on l1 should never show confirmed on ui
```
Comment (adamazad) [inline src/services/EcoBridge/Arbitrum/ArbitrumBridge.selectors.ts:122]: Can change this to an enum as well?

**Review by adamazad — CHANGES_REQUESTED** [review]
This is a good start! Few more concerns to address, and we can merge this. :)

**Review by adamazad — APPROVED** [review]
Awesome job! Let's merge this!

**Review by denciu — APPROVED** [review]

**Pull request merged**